libsolidity/interface/Natspec.cpp

```diff
@@ -225,7 +225,10 @@ Json::Value Natspec::extractCustomDoc(multimap<string, DocTag> const& _tags)
 	for (auto const& [tag, value]: _tags)
 		if (boost::starts_with(tag, "custom"))
 			concatenated[tag] += value.content;
-	Json::Value result;
+	// We do not want to create an object if there are no custom tags found.
+	if (concatenated.empty())
```

Comment:
@chriseth @cameel as a response to [this](https://github.com/ethereum/solidity/pull/11976#discussion_r712325924) added this explicit code.